CHANGELOG.md

```diff
@@ -25,6 +25,7 @@ and this project adheres to [Semantic Versioning](https://semver.org/spec/v2.0.0
 - URL options are now properly sent through to the server when doing a ClientTravel.
 - Fixed an issue that stopped the correct error message from being shown when the SchemaDatabase is missing.
 - Fixed an issue with `StartEditor.bat` not being generated correctly when building the server worker for local deployments outside of editor.
+- Replicated startup actors with NetLoadOnClient=false will no longer fail to load occasionally at startup
```

Comment:
As this is fixing something that was added in this version, we can omit the changelog update. Saves a merge also.

Reply:
reverted!